vcf-dumper/vcf-dumper-lib/src/test/java/embl/ebi/variation/eva/vcfdump/regionutils/RegionFactoryTest.java

```diff
@@ -88,4 +106,26 @@ public void divideRegionInChunks() {
         regions = regionFactory.divideRegionInChunks("1", 3000, 2500);
         assertTrue(regions.size() == 0);
     }
+
+    @Test
+    public void ifChromosomeWithNoCoordinatesInRegionFilterExecuteQueryForGettingMinAndMaxCoordinates() throws IOException, IllegalOpenCGACredentialsException {
```

Comment:
changed to executeQueryForGettingMinAndMaxCoordinatesWhenTheyAreNotIncludedInTheRegionFilter